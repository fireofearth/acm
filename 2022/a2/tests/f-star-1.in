6
4 0
4 1
3 1
10 2
6 2
7 3
