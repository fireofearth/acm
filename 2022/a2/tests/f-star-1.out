8
16
2
1304832
272
272
